src/Lykke.MatchingEngine.Connector/Helpers/MeConnectorServiceCollectionExtensions.cs

```diff
@@ -60,5 +60,30 @@ public static class MeConnectorServiceCollectionExtensions
             services.AddSingleton<IMatchingEngineClient>(s => s.GetRequiredService<TcpMatchingEngineClient>());
         }
 
+        /// <summary>
+        /// Registers <see cref="IMatchingEngineClient"/> in the <paramref name="services"/>
+        /// </summary>
+        /// <remarks><see cref="ILogFactory"/> should be registered in the container.</remarks>
+        /// <param name="services">Autofac container builder</param>
+        /// <param name="ipEndPoint">ME IP endpoint</param>
+        /// <param name="ignoreErrors">Flag indicating, if unknown response error should not rise exception</param>
+        public static void RegisterMeClient(
+            this IServiceCollection services,
+            IPEndPoint ipEndPoint,
+            bool ignoreErrors = false)
```

Comment:
Either make this parameter not optional, or remove RegisterMeClient overload at line 48. Otherwise calling RegisterMeClient with only two parameters will be ambigous.